seed/seed.json

```diff
@@ -34,14 +34,14 @@
             "experiments": [
                 {
                     "name": "Enabled",
-                    "probability_weight": 25,
+                    "probability_weight": 50,
                     "feature_association": {
                         "enable_feature": ["BraveAdblockDefault1pBlocking"]
                     }
                 },
                 {
                     "name": "Disabled",
-                    "probability_weight": 75,
+                    "probability_weight": 50,
```

Comment:
Wait, aren't we supposed to roll-out the `Disabled` state? 
The feature is enabled by default in the code @antonok-edm @pes10k